tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -360,6 +360,84 @@ exports[`mixed-4.php 1`] = `
 
 `;
 
+exports[`mixed-5.php 1`] = `
+<section class="<?= $sectionClass ?>">
+  <header class="<?= $headerClass ?>">
+    <?php
+
+      echo $this->element('collection/item-title', [
+        'class' => $itemTitleClass,
+        'small_title' => $item->small_title,
+        'title' => $item->title,
+        'url' => $url,
+        'color' => $item->title_color ?: 'primary',
+      ]);
+
+    ?>
+  </header>
+</section>
+
+<section class="<?= $sectionClass ?>">
+  <header class="<?= $headerClass ?>">
+    <?php
+
+      echo $this->element('collection/item-title', [
+        'class' => $itemTitleClass,
+        'small_title' => $item->small_title,
+        'title' => $item->title,
+        'url' => $url,
+        'color' => $item->title_color ?: 'primary',
+      ]);
+
+    ?>
+  </header> <?= $someOtherVar()->element('collection/item-title', [
+    'class' => $itemTitleClass,
+    'small_title' => $item->small_title,
+    'title' => $item->title,
+    'url' => $url,
+    'color' => $item->title_color ?: 'primary',
+  ]); ?>
+</section>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<section class="<?php echo $sectionClass; ?>">
+  <header class="<?php echo $headerClass; ?>">
+    <?php echo $this->element(
+        'collection/item-title',
+        [
+            'class' => $itemTitleClass,
+            'small_title' => $item->small_title,
+            'title' => $item->title,
+            'url' => $url,
+            'color' => $item->title_color ?: 'primary'
+        ]
+    ); ?>
+  </header>
+</section>
+
+<section class="<?php echo $sectionClass; ?>">
+  <header class="<?php echo $headerClass; ?>">
+    <?php echo $this->element(
+        'collection/item-title',
+        [
+            'class' => $itemTitleClass,
+            'small_title' => $item->small_title,
+            'title' => $item->title,
+            'url' => $url,
+            'color' => $item->title_color ?: 'primary'
+        ]
+    ); ?>
```

Comment:
We should use there `group`.

I.E. if it is allow be in one line print one line:
```php
<?php echo 1; ?>
```

If something break - break all (add newline after `<?php` and before `?>`) (as we do in prettier :smile: ):
```php
<?php
echo $this->veryVeryVeryVeryVeryVeryVeryLongFunction(
    'something',
   [
      'class' => $itemTitleClass
      // etc
   ]
);
?>
```

Reply:
@evilebottnawi I've been digging into this and I'm unsure if its going to be possible...

Inline nodes and regular php nodes are stored next to each other in a flat list, there's no top level node to group. ie if you had something like this

```php
<div>Foo</div>
<?php
$a = 1;
$some_otherThing = 1;
?>
<span>test</span>
```

the node list ends up being

```
[
  {type: "inline" ...},
  {type: "assign" ...},
  {type: "assign" ...},
  {type: "inline" ...}
]
```

So if we wanted to group the php open/close tags with the assign nodes, there's no parent node to actually group them together. Any ideas?

Reply:
We could maybe do something hacky our parser file to look through the ast we get from `php-parser` and create a faux top-level node for anything surrounded by inline nodes? Thats the only think I can think of atm

Reply:
@mgrip Let's solve this in `3. Improve group for <?php and ?>`. Right now let's left this as is

Reply:
sounds good 👍